src/app/src/components/card/Card.jsx

```diff
@@ -17,7 +18,9 @@ export default function Card({num}) {
       />
       <div className="content">
         <header>
-          <h2>Game Title #{num}</h2>
+          <h2>
+            {game.title} #{game.id}
```

Comment:
Is the game.id for debugging? I'm totally cool with keeping it if it is.

Reply:
Yes, it was for debugging and seeing if the items match with db. I must have just gotten use to seeing the ID and forgot it was not part of the original design. Probably remove it at some point.